src/scss/06_components/paragraphs/_hero.scss

```diff
@@ -70,6 +70,31 @@ $koro-height: 50px;
   }
 }
 
+[dir='rtl'] .hero__arrow {
+  left: auto;
+  right: $spacing-and-half;
+
+  @include breakpoint($breakpoint-l) {
+    right: $spacing-double;
+  }
+}
+
+// Hero image author styles.
+.hero--image-author__container {
+  position: relative;
+}
+
+.hero__image-author {
+  position: absolute;
```

Comment:
This `position: absolute` can be dangerous. What's the maximum limit for photographer field? I mean, it could go bad sideways if there's a long photographer info.
![image](https://github.com/user-attachments/assets/984b08db-c1ca-4167-b152-00d5ffcd7851)